deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -298,7 +298,7 @@ spec:
                   privileged: false
                   readOnlyRootFilesystem: true
                   runAsNonRoot: true
-                  runAsUser: 1001
+                  runAsUser: 1000550000
```

Comment:
does the user have to be explicitly set? I thought OCP can automatically assign the user